lib/ViewModels/CatalogItemControl.js

```diff
@@ -1,135 +0,0 @@
-'use strict';
```

Comment:
It's technically a breaking change to remove this.  Since the new `CatalogMemberControl` is compatible (as far as I can see), you can just rig this module to return that one.  See `AusGlobeViewer.js` for an example.  The `deprecationWarning` you see there is a good idea here, too.